prboom2/src/gl_clipper.c

```diff
@@ -350,10 +351,11 @@ static angle_t gld_FrustumAngle(void)
 
   // ok, this is a gross hack that barely works...
   // but at least it doesn't overestimate too much...
-  floatangle = 2.0f + (45.0f + (tilt / 1.9f)) * (float)gl_render_fov * ratio_scale / gl_render_multiplier / 90.0f;
-  a1 = (angle_t)xs_CRoundToInt(ANG1 * floatangle);
-  if (a1 >= ANG180)
+  fov = atan(1 / projMatrix[0]) * 360 / M_PI;
```

Comment:
There are a lot of places in the code base that use the `gl_render_fov` variable. Is this change going to cause any inconsistencies?

Also, is the comment here still the case? 😄

Reply:
This only affects seg culling, and using the GL projection matrix should prevent it from getting out of sync with actual rendering.

The actual horizontal render FOV is almost never `gl_render_fov`.  It's derived in a very roundabout manner by `M_ChangeFOV` and `R_SetupMatrix`.  This doesn't attempt to make the derivation any more sane.

The comment here is still correct, it's a very ad-hoc heuristic for the effective 2D view angle taking into account view pitch.